### PR TITLE
Added section on overriding lp config in loop

### PR DESCRIPTION
**Pull request by pryce-turner**
Small enhancements to nested parallelization docs showing how one can override default workflow params

### CODE REVIEW

File: docs/user_guide/advanced_composition/nested_parallelization.md [hunk 1]
```diff
@@ -82,6 +82,23 @@ def multi_wf(l: typing.List[int], chunk: int) -> typing.List[int]:
     return level1(l=l, chunk=chunk)
 ```
 
+Overrides let you add additional arguments to the launchplan you are looping over in the dynamic. Here we add caching:
```
Comment (neverett) [inline docs/user_guide/advanced_composition/nested_parallelization.md:85]:
```suggestion
Overrides let you add additional arguments to the launch plan you are looping over in the dynamic. Here we add caching:
```

**Review by neverett — APPROVED** [review]
One small suggestion, otherwise LGTM!

**Comment by pryce-turner** [comment]
@neverett would you mind reviewing again please since adding that suggestion voided the first one..

**Review by neverett — DISMISSED** [review]

**Comment by pryce-turner** [comment]
@neverett is this the same failure we saw last time? Do you know how to get past it?

**Comment by neverett** [comment]
@pryce-turner I think if you merge master into your branch, that should fix the error, since it was fixed in this PR: https://github.com/flyteorg/flyte/pull/5215 -- let me know if it's not fixed after that, though!

**Comment by pryce-turner** [comment]
@neverett looks like it dismissed your review again.. I actually fixed the code a bit so that makes sense. The failing check _should_ pass now though if you could please checkmark this one more time.

**Review by neverett — APPROVED** [review]

**Pull request merged**